Sources/StreamVideo/CallKit/CallKitAdapter.swift

```diff
@@ -20,6 +20,12 @@ open class CallKitAdapter {
         set { callKitService.iconTemplateImageData = newValue }
     }
 
+    /// The callSettings to use when joining a call (after accepting it on CallKit)
+    /// default: nil
```

Comment:
what's up with this  character?

Reply:
The change isn't showing. What was the issue?